bookOverview/src/main/kotlin/voice/bookOverview/bottomSheet/BottomSheetViewModel.kt

```diff
@@ -23,17 +23,16 @@ class BottomSheetViewModel
   internal fun bookSelected(bookId: Book.Id) {
     this.bookId = bookId
     scope.launch {
-      val items = viewModels.flatMap { it.items(bookId) }
-        .toSet()
-        .sorted()
+      val items = viewModels.sortedBy { it.menuOrder }
```

Comment:
Following YAGNI I'd rely on the order of the enum entries until a more smart ordering is needed.